experimental/listener.go

```diff
@@ -6,6 +6,24 @@ import (
 	"github.com/tetratelabs/wazero/api"
 )
 
+// StackIterator allows iterating on each function of the call stack, starting
```

Comment:
Mention somewhere that this is re-used for all calls with this listener?

Reply:
Should this be an implementation detail though? or do you mean to warn the user to not rely on keeping a pointer around?

Reply:
Added a comment on `Before()`.